roles/docker_traefik/templates/traefik.yml.j2

```diff
@@ -72,6 +72,22 @@ services:
         published: 443
         mode: ingress
     volumes:
-      - "{{ traefik__acme_dir }}:/etc/traefik"
-      - "{{ traefik__log_dir }}:/var/log/traefik"
+      - type: volume
```

Comment:
This only applies when `traefik__use_nfs` is true. 
What about when it is false?